docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.3'
+
+services:
+  web:
+    image: expense-tracker-group1
+    container_name: expense-tracker-container-new
```

Comment:
We are missing the environments that will contain the DB information such as username, password and the name of the DB.
Additionally, we need to create a volume for our database.

Reply:
environments and volumes completed